### PR TITLE
add optional flag and associated clear action

### PR DESCRIPTION
**Pull request by peller**
(c) Copyright IBM Corp. 2016

### CODE REVIEW

**Comment by peller** [comment]
fixes #76

File: urth-viz-explorer.html [hunk 1]
```diff
@@ -134,10 +134,20 @@
...
+            }
+
             paper-card-collapse {
                 display: block;
             }
 
+            .clear-option-btn {
+                color: #9b9b9b;
+                width: 2.5em;
+                height: 2.5em;
```
Comment (dalogsdon) [inline urth-viz-explorer.html]: There is a bit much padding on the delete button that causes the row height to be a bit higher. You can see the effect if there are 2 optional inputs in a row. It is simple to fix, just use similar styling to how I did the sort direction button, but looks like we need slightly different values. Try this here:

```css
    padding: 0;
    width: 1.3em;
    height: 1.3em;
```

**Comment by dalogsdon** [comment]
Looks good except one thing, see code comment.

File: urth-viz-explorer.html (hunk 1 above)
Comment (peller) [inline urth-viz-explorer.html]: done.

Comment (dalogsdon) [inline urth-viz-explorer.html]: looking good

**Pull request merged**